### PR TITLE
{submission model + status enum} definition

### LINKED ISSUE #5: Model: Submission
**The model should contain the following:**

- link - Link[]? (optional, if provided, the submission is a change request) (align with team/person assigned to that)
- linkName - string
- description - string
- status - enum (string): IN_REVIEW, APPROVED, DECLINED
- adminComment - string

**Acceptance Criteria:**
- Prisma scheme is created and is usable

### PR DESCRIPTION
**Pull request by mozsarmate**

### CODE REVIEW

File: packages/backend/prisma/schema.prisma [hunk 1]
```diff
@@ -2,27 +2,40 @@
...
-  id    String @id @default(uuid())
+    id String @id @default(uuid())
 }
 
 model Submission {
-  id    String @id @default(uuid())
+    id           String            @id @default(uuid())
+    link         Link              @relation(fields: [linkId], references: [id])
+    linkId       String
+    linkName     String
```
Comment (KostyalBalint) [inline packages/backend/prisma/schema.prisma:45]: why the linkName property is on the submission model?
Shouldn't it be in the link itself?

**Review by KostyalBalint — CHANGES_REQUESTED** [review]

File: packages/backend/prisma/schema.prisma (hunk 1 above)
Comment (mozsarmate) [reply packages/backend/prisma/schema.prisma:45]: good call, (linkName property was mentioned in the "model should contain:" section of the issue and the link itself was optional (included when the submission is an edit request))
but then, i guess, the submission should always contain a link, even if it is a create request

Comment (berenteb) [reply packages/backend/prisma/schema.prisma:45]: Read the user story: If the link property is defined, the submission is change request, and the linkname + description contains the new information. Obviously, if you have better solution, you can bring that here. If no link is connected, the linkname + description will be the new link entity's fields.

**Review by DannySS123 — APPROVED** [review]

**Review by KostyalBalint — APPROVED** [review]
Lgtm

**Pull request merged**